src/darjeeling/config.py

```diff
@@ -201,6 +207,11 @@ def err(m: str) -> NoReturn:
         elif threads is None:
             threads = 1
 
+        if 'run_redundant_tests' in yml:
+            if not isinstance(yml['run_redundant_tests'], bool):
+                err("'threads' property should be an bool")
+            run_redundant_tests = yml['run_redundant_tests']
```

Comment:
Let's change `run_redundant_tests` to `run-redundant-tests` to be consistent with the other YAML fields.

Reply:
Ah, good catch.